### PR TITLE
Level roles

### LINKED ISSUE #27: Giving out and replacing level roles
I would like the bot to be able to give out roles based on a member's level. Currently, I have roles for Level 1, and then multiples of 5. We can add more if we'd like. The purpose is to keep the most active people visible on the sidebar. Essentially, I'd like the bot to give "@Level 5" when someone reaches level 5, and take away their previous Level role. If possible, it would also be nice to have the bot give out the roles to everyone who doesn't have them right now, as doing that manually would be a pain.

### PR DESCRIPTION
**Pull request by EarlJr53**
Sam and I wrote the level roles stuff. Here it be.

### CODE REVIEW

**Comment by samollari** [comment]
Closes #27

File: triggers/triggers/levels.ts [hunk 1]
```diff
@@ -19,9 +20,52 @@ module.exports = <TriggeredCommand> {
...
+                    .sort((a:Role, b:Role) => getLevelNumberFromRole(a) - getLevelNumberFromRole(b))
+                    .last();
+                
+                member.roles.cache.forEach(role =>
+                    (newRole.name.startsWith('Level') && role.id != newRole.id) ?
+                        member.roles.remove(role)
+                      : null
+                );
+                
+                member.roles.add(newRole);
```
Comment (samollari) [inline triggers/triggers/levels.ts]: Try pulling this out into `util/levels.ts` and import it from there? It should also run that on every start up to update roles if it fell behind

**Review by samollari — DISMISSED** [review]

**Comment by EarlJr53** [comment]
As I mentioned in the latest commit, we still need to implement setLevelRoles on startup before we can close #27 / #34 . I wasn't exactly sure how we wanted to do that, so I committed what I had and we can figure that out soon.

**Comment by EarlJr53** [comment]
@i-am-gizm0 can we merge this into master for now before we add the on-startup check? And then do that after?

**Review by samollari — APPROVED** [review]

**Pull request merged**